go/libraries/doltcore/sqle/testutil.go

```diff
@@ -29,12 +29,14 @@ import (
 	"github.com/dolthub/dolt/go/libraries/doltcore/env"
 )
 
-// Executes all the SQL non-select statements given in the string against the root value given and returns the updated
-// root, or an error. Statements in the input string are split by `;\n`
+// ExecuteSql executes all the SQL non-select statements given in the string against the root value given and returns
+// the updated root, or an error. Statements in the input string are split by `;\n`
 func ExecuteSql(dEnv *env.DoltEnv, root *doltdb.RootValue, statements string) (*doltdb.RootValue, error) {
-	db := NewBatchedDatabase("dolt", dEnv.DbData())
-	engine, ctx, err := NewTestEngine(context.Background(), db, root)
+	db := NewDatabase("dolt", dEnv.DbData())
 
+	engine, ctx, err := NewTestEngine(context.Background(), db, root)
+	DSessFromSess(ctx.Session).EnableBatchedMode()
+	err = ctx.Session.SetSessionVariable(ctx, sql.AutoCommitSessionVar, uint8(0))
```

Comment:
Similar to the comment earlier, for all of the `SetSessionVariable` calls you can just pass in `false` as it's clearer.